ts/components/messages/MessageListComponent.tsx

```diff
@@ -67,11 +68,13 @@ class MessageListComponent extends React.Component<Props> {
       paymentByRptId,
       ListEmptyComponent
     } = this.props;
-
     const refreshControl = (
       <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
     );
 
+    messagesToRead = messages.filter(obj => !obj.isRead).length;
+    alert(messagesToRead);
```

Comment:
what's this alert?

Reply:
Sorry it was only for tests and I forgot to remove it